benchmarks/odl_text_runtime.py

```diff
@@ -10,6 +10,7 @@
 import spams
 from sklearn.feature_extraction.image import extract_patches_2d
 
+from dicodile._config import DATA_HOME
```

Comment:
I would not import from private files outside of the lib. why not calling this config.py without the _ ?

Reply:
It was a quick decision thinking that config need not be public, I'm fixing it.